src/batou_ext/ssl.py

```diff
@@ -339,9 +337,9 @@ def configure(self):
 
         self += batou.lib.file.File(
             "cert_check_{}.sh".format(self.name),
-            content=pkg_resources.resource_string(
-                __name__, "resources/ssl/local_certificate_check.sh"
-            ),
+            content=(
+                files("batou_ext") / "resources/ssl/local_certificate_check.sh"
```

Comment:
is there any specific reason to not use `__name__` here, but hardcode `batou_ext` instead?

Reply:
In this context, `__name__` isn't available. The name is fixed, so hard-coding it looked like a reasonable fix.

Reply:
Huh. That does't make sense. `__name__` is the module name, see https://docs.python.org/3/reference/datamodel.html#module.__name__ so would be `batou_ext.ssl` here. `files()` behaves differently so not using `__name__` is easier. The non hard-coded variant is `__spec__.parent` (which is `"batou_ext"`)